geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExportStackTraceCommand.java

```diff
@@ -50,6 +53,10 @@
   public static final String STACK_TRACE_FOR_MEMBER = "*** Stack-trace for member ";
   private final GetStackTracesFunction getStackTracesFunction = new GetStackTracesFunction();
 
+  DateTimeFormatter formatter =
+      DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss.SSS", Locale.UK)
```

Comment:
Why are you hardcoding the `Locale` to `UK` here?.
I think you should use `Locale.getDefault()` or [`DateTimeFormatter.ofPattern(String pattern)`](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ofPattern-java.lang.String-) directly.

Reply:
Yes we can set it getDefault, I didn't see it. I just pick UK to be consistent with timeStamp in log file. I will change it. Thank for  the comment!